stock-filters/ChirpVillage/Surface.py

```diff
@@ -78,8 +78,8 @@ def get_height(self, x, z, level):
         :return: height (y-value) of the first ground block before air.
         """
         print("get_height")
-        top = self.y_start + 40
-        bottom = self.y_start - 40
+        top = self.y_start + 60
+        bottom = self.y_start - 60
```

Comment:
we might want to adjust this to be like if y_start > y_end then top is ystart + 40 and bottom is yend - 40 else top is yend+40 and bottom is ystart-40. This will create a bigger search space but there is a greater probability of the surface block being within the selection with 40 buffer on either side

Reply:
Ill fix